float64 pdop
float64 hdop
float64 vdop
